src/notation/view/pianokeyboard/pianokeyboardpanelcontextmenumodel.cpp

```diff
@@ -71,6 +73,40 @@ int PianoKeyboardPanelContextMenuModel::numberOfKeys() const
     return configuration()->pianoKeyboardNumberOfKeys().val;
 }
 
+bool PianoKeyboardPanelContextMenuModel::pianoKeyboardNotatedPitch() const
+{
+    return configuration()->pianoKeyboardNotatedPitch().val;
+}
+
+MenuItem* PianoKeyboardPanelContextMenuModel::makePitchMenu()
+{
+    MenuItemList items;
+
+    std::vector<std::pair<muse::TranslatableString, bool> > possibleNotationStates {
+        { muse::TranslatableString("notation", "Switch to Notated Pitch"), true },
+        { muse::TranslatableString("notation", "Switch to Playback Pitch"), false },
```

Comment:
```suggestion
        { muse::TranslatableString("notation", "Use notated pitch"), true },
        { muse::TranslatableString("notation", "Use sounding pitch"), false },
```
Note that we use "Sentence case" and not "Title Case".